src/docs/user/references/DataConnectors.adoc

```diff
@@ -291,3 +291,83 @@ You have two options for providing your login credentials:
 * Instead of hardcoding the name of your output file in your U-SQL script, you can use the placeholder `OUTPUT_FILE`, which is automatically replaced by the value of `AZ_DLA_OUTPUT`.
 * Once downloaded, the output file will be stored in your *User Space* (and _not_ in the Global Space).
 ====
+
+=== ERP
+
+Proactive's ERP connectors enable communication and data exchange with popular ERP software providers.
+
+==== SAP ECC
+
+This connector allows you to interact with an SAP ECC system (not S/4HANA) by means of remote function calls (RFC).
+
+*Variables:*
+
+.SAP ECC Connector variables
+[cols="2,5,2,2,2,2"]
+|===
+| *Variable name* | *Description* | *Level* | *Required?* | *Type*  | *Default/Examples*
+| `JCO_ FUNCTION`
+| the name of a remote-enabled function module on your SAP System. You can verify the available functions in your ECC transaction SE37.
+| Workflow
+| Yes
+| String
+| e.g. RFC_SYSTEM_INFO
+|===
+
+*How to use this connector:*
+
+In order to securely connect to your SAP ECC system, you first need to store your SAP logon information as ProActive's link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials]. The required credentials are:
+
+.SAP connection credentials
+[cols="1,3,1"]
+|===
+| *Credential* | *Description* | *Example*
+| `JCO_ASHOST`
+| Application server host IP.
+| 10.10.10.1
+| `JCO_SYSNR`
+| System number.
+| 00
+| `JCO_CLIENT`
+| Client number.
+| 100
+| `JCO_USER`
+| SAP logon user name.
+| myuser
+| `JCO_PASSWD`
+| SAP logon password (it will be hidden).
+| mypassword
+| `JCO_LANG`
+| Preferred language.
+| EN
+| `JCO_SAPROUTER`
+| _(Optional)_ An SAP Router string, in case you are connecting from an external
+               network. It is important that you keep the /H/ tags.
+| /H/192.168.0.1/H/
+|===
+
+Once the credentials are set, the workflow can be executed. If the connection is successful, the connection attributes will be listed as output. *NB.* Proactive's SAP connector provides libraries to connect to an SAP server from a 64-bit OS (Windows, Linux, MacOS); libraries for other OS can be obtained through SAP Marketplace.
```

Comment:
to connect to **a** SAP server

Reply:
I had some doubt but I think "an SAP server" is correct, as it's read "an ess-a-pee server"
http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html